test/fixtures/test-runner/output/skip_pattern.js

```diff
@@ -0,0 +1,20 @@
+// Flags: --test-skip-pattern=disabled --test-skip-pattern=/no/i
```

Comment:
Are there any tests with `--test-name-pattern` and `--test-skip-pattern` interacting? If not, please add some.